### DISCUSSION TITLE
npm run build | sh: 1: next not found

### DISCUSSION THREAD

**Discussion by phyouthcenter1** [vercel/next.js#53450] [Help]
### Summary

I am getting the error that next is not found. I have tried installing globally, reinstalling, deleting package-lock.json and re running npm install and nothing works. My build script is `next build`.

![image](https://github.com/vercel/next.js/assets/67077196/676bc743-66cd-4c80-b3ec-db2b53296b38)

### Additional information

```js
I can not run any command starting with next because it says "Command 'next' not found, but can be install with:

apt install mailutils-mh
apt install mmh
apt install nmh
"
```

**Comment by icyJoseph** [comment]
how is your project structured? you are running npm run build from within `tovy` but is next.js installed there?

**Comment by bloodykheeng** [comment]
@phyouthcenter1 

@SahDhiraj  the solution is easy as eating kfc 
you just need to enable  permisiions 

for my case is did this
Run this command to change the owner and group of /var/www/cfp to root
```
sudo chown -R root:root /var/www/cfp

```


NB first type whoami in termial to know who u are and replace root with who u are 
some times whoami shows u ubuntu its ubuntu 

```
sudo chown -R ubuntu:ubuntu /var/www/cfp
```

then after u install node modules with npm install

the error u got means next node module isn't installed yet
